core/src/main/resources/lib/form/helpLink.jelly

```diff
@@ -54,12 +54,12 @@ THE SOFTWARE.
   <j:choose>
     <j:when test="${attrs.url!=null}">
       <j:set var="altText" value="${attrs.featureName != null ? '%Help for feature:' + ' ' + attrs.featureName : '%[Help]'}" />
-      <td class="setting-help">
+      <span class="setting-help">
         <a href="#" class="help-button" helpURL="${rootURL}${attrs.url}"><l:icon class="icon-help icon-sm" alt="${altText}"/></a>
-      </td>
+      </span>
     </j:when>
     <j:otherwise>
-      <td class="setting-no-help" />
+      <span class="setting-no-help" />
```

Comment:
span tags are not self-closing tags, I think this should be `<span class="setting-no-help"></span>`